src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -96,9 +103,8 @@ public static partial class RequestDelegateFactory
     /// <param name="handler">A request handler with any number of custom parameters that often produces a response with its return value.</param>
     /// <param name="options">The <see cref="RequestDelegateFactoryOptions"/> used to configure the behavior of the handler.</param>
     /// <returns>The <see cref="RequestDelegateResult"/>.</returns>
-#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
+    [RequiresUnreferencedCode("RequestDelegateFactory performs object creation, serialization and deserialization on the delegates and its parameters. This cannot be statically analyzed.")]
```

Comment:
Since this can't be trimmed, should MapGet and friends that use it also be marked with `RequiresUnreferencedCode`? (or haven't they been annotated yet because they're in a different assembly?)

Reply:
Yup. I think those are in the routing assemblies, so we should start seeing warnings there once they are configured for trimming.